### DISCUSSION TITLE
Onboarding error 500

### DISCUSSION THREAD

**Discussion by Kohout157** [renovatebot/renovate#26823] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

Installing renovate bot into an organization throws the following error:
![image](https://github.com/renovatebot/renovate/assets/71974700/c6584b8a-d7ff-4780-98ae-4242c5e9072e)

The organization is visible in the organization list, but when I start onboarding it throws the error again. I tried making new organizations but they all work, including my already existing ones, only this one is stuck.

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
It appears that there's two orgs with the same name in our backend DB. Do you know if this org existed previously, was deleted, then recreated?

**Reply by Kohout157** [reply]
Yep, this is exactly the case, even one owner is the same as previously, can it be fixed?

**Reply by rarkins** [reply]
Yes, we'll fix it manually for now so you can move ahead

**Reply by rarkins** [reply, marked as answer]
We've disabled the old org in our backend DB so please try again
